cfme/scripting/bz.py

```diff
@@ -110,6 +116,51 @@ def report(directory):
     get_report(directory)
 
 
+@main.command(help="Generate BZ report on bugs going into a z-stream")
+@click.option(
+    "--stream",
+    "-s",
+    "stream",
+    default=current_version().vstring,
+    help="Stream for which to get BZs (e.g. 5.10.11.0)",
+    show_default=True
+)
+@click.option(
+    "--query-file",
+    "-q",
+    "query_file",
+    default="fixed_in_query.yaml",
+    help="Template query file",
+    show_default=True
+)
+@click.option(
+    "--output",
+    "-o",
+    "output",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Show information about BZs in the terminal"
+)
+@click.option(
+    "--filename",
+    "-f",
+    "fname",
+    default=f"bz-report-{current_version().vstring}.yaml",
+    help="Filename into which the report is written (must be yaml file)"
+)
+def build_report(stream, query_file, output, fname):
+    query_path = QUERY_PATH.join(query_file)
+    bz_data = BugzillaData(query_path, BZ_URL, "component", credential_file="None")
```

Comment:
As per my understanding. I we not logged in to `python-bugzilla` then more chances to miss `private` bug.

Reply:
Updated bugzilla-data and this PR to allow for this